src/Compilers/CSharp/Portable/Symbols/Source/ParameterHelpers.cs

```diff
@@ -98,6 +109,33 @@ internal static class ParameterHelpers
             return parameters;
         }
 
+        private static bool IsPropertyWithoutBody(Symbol owner)
+        {
+            if (owner is SourcePropertySymbol)
```

Comment:
> What is the case where this can happen? When can you have parameters on a property but it isn't a SourcePropertySymbol?

The code will be more robust in the face of future refactoring and/or changes in the hierarchy of implementation types if we don't depend on specific types unless it is necessary. It doesn't look like it is necessary in this case.